app/views/email/CaptureEmailView.scala.html

```diff
@@ -57,16 +57,23 @@ <h1 id="page-heading" class="heading-large">@messages("captureEmail.title")</h1>
         @if(!(currentEmail == "")) {
             <div class="form-group">
                 <details>
-                    <summary role="button"><span class="summary"></span>@messages("captureEmail.removeEmail.title")</summary>
+                    <summary role="button">
+                      <span class="summary"></span>
+                      @messages("captureEmail.removeEmail.title")
+                    </summary>
                     <div class="panel panel-border-narrow">
-                        <a target="_blank" href="@appConfig.contactHmrcUrl">@messages("captureEmail.removeEmail.contactUs")</a> @messages("captureEmail.removeEmail.description")
+                        <a target="_blank" href="@appConfig.contactHmrcUrl">
+                          @messages("captureEmail.removeEmail.contactUs")
+                        </a> @messages("captureEmail.removeEmail.description")
                     </div>
                 </details>
             </div>
         }
 
         <div id="hmrc-privacy-notice">
             @messages("hmrcPrivacyNotice.details")
-            <a target="_blank" href="@appConfig.hmrcPrivacyNoticeUrl">@messages("hmrcPrivacyNotice.link")</a>@messages("common.fullstop")
+            <a target="_blank" href="@appConfig.hmrcPrivacyNoticeUrl">
+              @messages("hmrcPrivacyNotice.link")
+            </a>@messages("common.fullstop")
```

Comment:
Putting the full stop message on a new line has added a whitespace between the sentence and the full stop. The fact that a unit test did not flag this up tells me we are missing a test for it. Could you also add a unit test to `CaptureEmailViewSpec` to check the privacy notice content.
<img width="592" alt="image" src="https://user-images.githubusercontent.com/29063729/87671633-94ca4980-c769-11ea-867f-631935898324.png">